build.gradle

```diff
@@ -1,16 +1,16 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext {
-        kotlin_version = '1.5.21'
-        compose_version = '1.0.1'
+        kotlin_version = '1.7.10'
+        compose_version = '1.3.0'
     }
 
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.2.2" // Be careful about upgrading this.
+        classpath 'com.android.tools.build:gradle:7.0.4' // Be careful about upgrading this.
```

Comment:
Just want to make sure we're okay with this - I couldn't get the app building without this upgrade

Reply:
Yup this is old enough that it's fine IMO